certbot/crypto_util.py

```diff
@@ -34,15 +36,19 @@
 
 
 # High level functions
-def init_save_key(key_size, key_dir, keyname="key-certbot.pem"):
+def init_save_key(
```

Comment:
I agree with the concept of what you're saying here, but forward looking when we support both RSA as well as ECDSA from a single invocation, we would need both, no?

If I wanted a 4096b RSA key as well as a 384b ECDSA key, I would need to specify both at runtime.  While we could get away with the singular form today, this would be a problem for the multi-use case later, and cause a backwards support problem?

I'd rather move the sub functions to the generic `key_size` usage, and keep the CLI argument set as both the standard `--key-size`, and new `--ecdsa-key-size`.

Thoughts?

Reply:
Well, I think `init_save_key` is also a low level function in the global workflow of issuing a certificate. Its purpose is focused and simple: it generates a key. Why not keeping things like that ?

Indeed, when `--key-path` will support multiple values, the general workflow could invoke two times `init_save_key`, once for the RSA, a second one for the ECDSA.

Anyway, a lot of refactoring will be necessary when we will work on multiple keys. So things like they are in this PR will evolve again. Until that, having one unique `key_size` parameter limits at its maximum the change in the public API.

Reply:
Ok, modified the code to re-factor ```init_save_key``` back to the single purpose low level function and follow your logic statement above, but I kept the separate ```ecdsa_key_size``` and ```rsa_key_size``` values on purpose.

This I believe will make subsequent PR's a bit easier to start on since the groundwork for keeping track of both key types/sizes is already in place.

Reply:
Ok, works for me.